impeller/entity/entity_pass_clip_stack.h

```diff
@@ -21,6 +23,16 @@ struct ClipCoverageLayer {
 ///        stencil buffer is left in an identical state.
 class EntityPassClipStack {
  public:
+  struct ReplayResult {
+    Entity entity;
+    std::optional<Rect> clip_coverage;
+  };
+
+  struct ClipStateResult {
+    bool should_render = false;
+    bool clip_did_change = false;
```

Comment:
should_render is sort of documemted on the method body, can you document what clip_did_change means?

Reply:
Done.